impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -644,6 +658,10 @@ define("common-ui/util/_a11y", ["./_focus"], function (focusUtil) {
       });
     }
 
+    function useTabsForNavigation(elem) {
+      return (elem && elem.dataset.penToolbarNavUseTabs) || null;
+    }
```

Comment:
Imo, this function would better return `boolean`. When the attribute is not there, should return `false`. When the attribute does not have the value `true`, should also return `false`.